### PR TITLE
added SheetCollection class

### PR DESCRIPTION
**Pull request by shoutoutloud**
Please have a look.Thanks.

### CODE REVIEW

**Comment by shoutoutloud** [comment]
Hi.I did what you said but it is still giving errors.

File: src/SheetCollection.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+class SheetCollection extends Collection
```
Comment (rap2hpoutre) [inline src/SheetCollection.php]: Remove extra space here  (at the end of the line)

File: src/SheetCollection.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
+class SheetCollection extends Collection 
+{
```
Comment (rap2hpoutre) [inline src/SheetCollection.php]: Remove extra space here (at the end of the line)

**Comment by rap2hpoutre** [comment]
Thank you! There was a problem with extra spaces at the end of the lines.

**Comment by shoutoutloud** [comment]
I did remove the extra spaces.What should I do now?

**Comment by rap2hpoutre** [comment]
Thank you! Merged!

**Comment by rap2hpoutre** [comment]
FYI, I just added a namespace and used the correct namespace here: https://github.com/rap2hpoutre/fast-excel/commit/2534cdd7fe0bc877c05d67db99d9b2da062ed203. Anyway, thank you!

**Pull request merged**